### PR TITLE
init response inside the app and reset each time

### PR DESCRIPTION
**Pull request by dselivanov**
According to my tests this PR improves perf ~20%. on inst/fib.R benchmark on 4 cores it serves ~10500 requests per second compared to ~8500 requests per second with dev branch

### CODE REVIEW

**Comment by artemklevtsov** [comment]
Nice hack.

File: R/RestRserveResponse.R [hunk 1]
```diff
@@ -85,6 +85,13 @@ RestRserveResponse = R6::R6Class(
       self$status_code = as.integer(status_code)
       self$context = new.env(parent = emptyenv())
     },
+    reset = function(content_type, serializer = NULL) {
```
Comment (artemklevtsov) [inline R/RestRserveResponse.R]: Need tests for this method. Also can added in #47.

**Comment by dselivanov** [comment]
I don't know why calling R6 constructor takes that long

ср, 7 авг. 2019 г., 17:54 Artem Klevtsov <notifications@github.com>:

**Comment by artemklevtsov** [comment]
What about other mutable fields in response?

**Comment by artemklevtsov** [comment]
See this post for the more information about perfomance: https://r6.r-lib.org/articles/Performance.html.

**Comment by dselivanov** [comment]
will be implemented in #71

**Pull request closed without merging**